keyboards/cutie_club/giant_macro_pad/readme.md

```diff
@@ -0,0 +1,13 @@
+# Cupar19 Giant Macro Pad
+
+![Cupar19 Giant Macro Pad](https://user-images.githubusercontent.com/15253132/118325863-0135e680-b4fc-11eb-99f2-754b155b0acd.png)
+
+* Keyboard Maintainer: [Cutie Club](https://github.com/cutie-club/)
+* Hardware Supported: STM32F072 based giant 20x20 macro pad
```

Comment:
If this is a one-off, shouldn't it go into `keyboards/handwired/`?

Reply:
Thank you for the response

"One off" is bad wording on my part, 5 exists, It just wasnt a group buy or anything, but a PCB commission.

I feel it would not fit in handwired so well since its not handwired?

What do you propose, I put instead for Hardware Availability? or maybe it would fit in the Handwired folder?

Reply:
All good for now, will raise some discussions with the other collaborators regarding this sort of thing.